extensions/amp-copy/0.1/amp-copy.js

```diff
@@ -0,0 +1,180 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {isLayoutSizeDefined} from '../../../src/layout';
+import {user} from '../../../src/log';
+import {CSS} from '../../../build/amp-copy-0.1.css';
+
+/** @const */
+const IOS_USER_AGENT = /ipad|ipod|iphone/i;
+
+/** @const */
+const COPY_SUCCESS = 'Copied!';
+
```

Comment:
This should be user input - no strings allowed in here since they will need to be translated. 

Include a success message in your format and show it of the author had included one else update a unicode ✓ mark on as the buttons text (change it back after a setTimeout interval)